tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -58,6 +61,8 @@ spec:
               echo "Max retries exceeded."
               exit 1
             fi
+            RESULTS_JSON=$(jq --arg name "$2" --arg url "$4:$5" '.images |= map(select(.name==$name).urls +=
```

Comment:
Question: Shouldn't we save also the urls that were skipped (already existing)? I guess it depends on what we're saving - you can say that we want to capture what was pushed. And the skipped ones were not pushed. But on the other hand, for the user this will be a good place (once shown in the UI) to get the resulting images with tags. I think it would be nice if the last PLR showed all of the pushed images - including whatever was already done in a previous PLR for the partial snapshot. WDYT?

Reply:
I think that is a good suggestion. I will update the code and tests in a bit and post here once I do

Reply:
Updated, including the digests-match test

Reply:
Cool!